kotlin/benchmark/src/jvmMain/resources/random_generated.json

```diff
@@ -0,0 +1,346 @@
+{
```

Comment:
Some of these files (especially `citm_catalog.json`) are pretty big, and bloat up the repo for everyone that uses FlatBuffers. Can we replace these with a script that wgets the files or whatever for whoever wants to run the benchmark?

Reply:
or just instructions where to get the files?

Reply:
Where should we keep it? 

The alternative is to compress all, it should save 75% of space.

Reply:
Dunno, where you get these files from? can we point to a url or repo?

Reply:
Almost all benchmarks on web uses those datasets, you can find it here:
https://github.com/miloyip/nativejson-benchmark/tree/master/data
or here
https://github.com/serde-rs/json-benchmark/tree/master/data

Reply:
that's what I meant. So either add a script that pulls those files as part of benchmarking, or a doc instructing benchmark users to get those files manually.

Reply:
If you are okay with pointing to an external repo, then we are good.